internal/commands/user/enable.go

```diff
@@ -96,3 +96,12 @@ func userEnableRow(output userOutput, row map[string]interface{}) {
 	row[headerEnabled] = enabled
 	row[headerDetails] = details
 }
+
+func (i enableInputs) resolveUsers(ui terminal.UI, realmClient realm.Client, app realm.App) ([]realm.User, error) {
+	found, foundErr := i.findUsers(realmClient, app.GroupID, app.ID)
+	if foundErr != nil {
+		return nil, foundErr
+	}
+
+	return i.selectUsers(ui, found, "enable")
```

Comment:
seems like this is repeated for all actions, if we're going to abstract out `selectUsers` and `findUsers`, why not just remove this `resolveUsers` func and call `selectUsers`?

Reply:
blame me for that one!  feel free to go with tim's route here